src/api/category/category.class.ts

```diff
@@ -0,0 +1,11 @@
+import mongoose from 'mongoose';
+import ICategory from './category.interface';
+
+mongoose.connect(process.env.MONGO_DB_URI as string);
```

Comment:
Why we need connection here ?

Reply:
@Shehanka has used `MongoClient` in MongoHelper. Since we use `Mongoose`, This is the connection that binds with the Mongoose Model instance.  I added this here because, If I change the `MongoHelper`, @Shehanka 's implementation will fail or other forks meet conflict.

Reply:
@banujan6 remove the mongoose connect from here. Don't change the structure.